src/providers/twitch/pubsubmessages/LowTrustUser.hpp

```diff
@@ -0,0 +1,66 @@
+#pragma once
+
+#include <magic_enum.hpp>
+#include <QColor>
+#include <QJsonObject>
+#include <QString>
+
+namespace chatterino {
+
+struct PubSubLowTrustUserMessage {
+    enum class Type {
+        NewMessage,
+        TreatmentUpdate,
+
+        INVALID,
+    };
+    QString typeString;
+    Type type = Type::INVALID;
+
+    QJsonObject data;
+
+    // This here is documented to be always available, but structure may differ
+    QJsonObject types;
+    QString banEvasionEvaluation;
+    QString treatment;
+    QString lowTrustID;
+    QString channelID;
+
+    QString updater_userID;
```

Comment:
Fix naming style, should be `updaterUserID`, same style goes for all variables & classes you've used underscores in